src/main/java/core/basesyntax/service/AmountFromFile.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface AmountFromFile {
+    List<Integer> getAmountInShop(List<String> fruits, List<FruitTransaction> fruitTransactions);
```

Comment:
The method 'getAmountInShop' should probably return a more complex type than just a List of Integers. It's not clear what these integers represent. It would be better to return a Map<Fruit, Integer> or a custom object that encapsulates the fruit and its quantity in the shop.